src/main/java/uk/gov/hmcts/darts/arm/config/UnstructuredToArmProcessorConfiguration.java

```diff
@@ -13,6 +13,7 @@
 @Configuration
 public class UnstructuredToArmProcessorConfiguration {
 
-    private int maxResultSize;
+    private int maxArmManifestItems;
```

Comment:
these dont exist, they are in a different place in the yaml. probably need to change line 10.
Also, max-result-size is still int eh yaml, and so should be removed.